.github/workflows/nodejs.yml

```diff
@@ -9,13 +9,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [6.x, 10.x, 12.x]
+        node-version: [10.x, 12.x]
```

Comment:
Yeah, that's a thing. There's still some stuff running 6 :(

Reply:
Sure, but they won't be getting new versions of this.

Reply:
in that case, please bump a version of the package. Practically this means dropping node version support, so maybe let's respect server and go major?

Reply:
Sure, done.